hack/make-release-artifacts.sh

```diff
@@ -25,46 +25,47 @@ if [[ ! -d "${bin_dir}" ]]; then
   exit 1
 fi
 
-krew_tar_archive="krew.tar.gz"
-krew_exe="krew.exe"
-
-# copy license
-cp -- "${SCRIPTDIR}/../LICENSE" "./${bin_dir}"
-
-# create a out/krew.exe convenience copy
-if [[ -x "./${bin_dir}/krew-windows_amd64.exe" ]]; then
-  cp -- "./${bin_dir}/krew-windows_amd64.exe" "./out/krew.exe"
+checksum_cmd="shasum -a 256"
+if hash sha256sum 2>/dev/null; then
+  checksum_cmd="sha256sum"
 fi
+checksum_sed=""
 
-# consistent timestamps for files in bindir to ensure consistent checksums
 while IFS= read -r -d $'\0' f; do
-  echo "modifying atime/mtime for $f"
-  TZ=UTC touch -at "0001010000" "$f"
-  TZ=UTC touch -mt "0001010000" "$f"
-done < <(find "${bin_dir}" -print0)
+  archive_dir="$(mktemp -d)"
+  cp "$f" "${archive_dir}"
+  cp -- "${SCRIPTDIR}/../LICENSE" "${archive_dir}"
+  archive="$(basename "$f" .exe).tar.gz"
+  echo >&2 "Creating ${archive} archive."
+  (
+    cd "${archive_dir}"
+    # consistent timestamps for files in archive dir to ensure consistent checksums
+    TZ=UTC touch -amt "0001010000" ./*
```

Comment:
I imagine this could make trouble on Mac/BSD, because the man says

> -a     change only the access time
> -m     change only the modification time

So I'm worried they take the _only_ too seriously. If you can check it works then great.

However, we can still make the change, because nowadays the release is only done by GH  actions and they run on Linux where I have verified that the result is the same.

Reply:
The doc start with 
> Update the access and modification times of each FILE to the current time.
So I guess the *only* means that this option will change this one only compared to both.
And at the end, using both options is same as using none...

Reply:
:D good catch